\hypertarget{classpip_1_1__vendor_1_1pep517_1_1in__process_1_1__in__process_1_1___dummy_exception}{}\doxysection{pip.\+\_\+vendor.\+pep517.\+in\+\_\+process.\+\_\+in\+\_\+process.\+\_\+\+Dummy\+Exception Class Reference}
\label{classpip_1_1__vendor_1_1pep517_1_1in__process_1_1__in__process_1_1___dummy_exception}\index{pip.\_vendor.pep517.in\_process.\_in\_process.\_DummyException@{pip.\_vendor.pep517.in\_process.\_in\_process.\_DummyException}}


Inheritance diagram for pip.\+\_\+vendor.\+pep517.\+in\+\_\+process.\+\_\+in\+\_\+process.\+\_\+\+Dummy\+Exception\+:
% FIG 0


Collaboration diagram for pip.\+\_\+vendor.\+pep517.\+in\+\_\+process.\+\_\+in\+\_\+process.\+\_\+\+Dummy\+Exception\+:
% FIG 1


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Nothing should ever raise this exception\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/pip/\+\_\+vendor/pep517/in\+\_\+process/\+\_\+in\+\_\+process.\+py\end{DoxyCompactItemize}
